Replace "Mesh Expansion" with "VM Support" and related edits. (#5215)
istio/istio.io: content/en/docs/examples/virtual-machines/bookinfo/index.md

---
title: Bookinfo with a Virtual Machine
description: Run the Bookinfo application with a MySQL service running in a virtual
  machine within your mesh.
weight: 60
keywords:
- virtual-machine, vms
aliases:
- /docs/examples/integrating-vms/
- /docs/examples/mesh-expansion/bookinfo-expanded
- /docs/examples/vm-bookinfo
---

> This example deploys the Bookinfo services across Kubernetes with one

Reviewer: ```suggestion
This example deploys the Bookinfo application with one
```

> virtual machine (VM), and illustrates how to use Istio service mesh to control

Reviewer: ```suggestion
service running on a virtual machine (VM), and illustrates how to control
```

## Overview

{{< image width="80%" link="./vm-bookinfo.svg" caption="Bookinfo running on VMs" >}}

<!-- source of the drawing
https://docs.google.com/drawings/d/1G1592HlOVgtbsIqxJnmMzvy6ejIdhajCosxF1LbvspI/edit
 -->

## Before you begin

- Setup Istio by following the instructions in the
  [Installation guide](/docs/setup/install/kubernetes/).

- Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application (in the `bookinfo` namespace).

> - Create a VM named 'vm-1' in the same project as Istio cluster, and [Join the Mesh](/docs/examples/virtual-machines/single-network/).

Reviewer 2: ```suggestion
- Create a VM named 'vm-1' in the same project as the Istio cluster, and [join the Mesh](/docs/examples/virtual-machines/single-network/).
```

Reviewer 2: Is the use of "project" correct here? Thats a GCP-ism.

Reviewer: This whole example is Google-specific. Rigs and I agreed to just fix the terminology for now, but after 1.4 somebody should rework the example to work on any platform.
